$NetBSD: patch-CMakeLists.txt,v 1.4 2020/07/21 07:31:28 adam Exp $

Remove the weird hack.
Do not look for Doxygen.
Support GLUT/glut.h.

--- CMakeLists.txt.orig	2020-07-20 13:56:40.000000000 +0000
+++ CMakeLists.txt
@@ -57,11 +57,7 @@ set(JAS_SO_MINOR 0)
 set(JAS_SO_RELEASE 0)
 # This is a temporary hack for OSX that should be removed when JAS_SO_VERSION
 # is next incremented.
-if (MACOSX)
-set(JAS_SO_NAME "5.${JAS_SO_MINOR}.${JAS_SO_RELEASE}")
-else()
 set(JAS_SO_NAME "${JAS_SO_VERSION}.${JAS_SO_MINOR}.${JAS_SO_RELEASE}")
-endif()
 
 message("Shared library ABI version: ${JAS_SO_VERSION}")
 message("Shared library build version: ${JAS_SO_NAME}")
@@ -221,7 +217,6 @@ endif()
 # Perform plaform checks.
 ################################################################################
 
-find_package(Doxygen)
 find_package(LATEX COMPONENTS PDFLATEX)
 find_program(BASH_PROGRAM bash)
 
@@ -310,11 +305,13 @@ if (JAS_ENABLE_OPENGL AND OPENGL_FOUND)
 		message("GLUT include directory: ${GLUT_INCLUDE_DIR}")
 		message("GLUT libraries: ${GLUT_LIBRARIES}")
 		set(CMAKE_REQUIRED_INCLUDES ${GLUT_INCLUDE_DIR})
+		check_include_files(GLUT/glut.h JAS_HAVE_GLUT_GLUT_H)
 		check_include_files(GL/glut.h JAS_HAVE_GL_GLUT_H)
 		check_include_files(glut.h JAS_HAVE_GLUT_H)
 		message("JAS_HAVE_GLUT_H: ${JAS_HAVE_GLUT_H}")
 		message("JAS_HAVE_GL_GLUT_H: ${JAS_HAVE_GL_GLUT_H}")
-		if (JAS_HAVE_GL_GLUT_H OR JAS_HAVE_GLUT_H)
+		message("JAS_HAVE_GLUT_GLUT_H: ${JAS_HAVE_GL_GLUT_H}")
+		if (JAS_HAVE_GLUT_GLUT_H OR JAS_HAVE_GL_GLUT_H OR JAS_HAVE_GLUT_H)
 			set(JAS_HAVE_OPENGL 1)
 			include_directories(${GLUT_INCLUDE_DIR} ${OPENGL_INCLUDE_DIR})
 		else()
